egs/callhome_diarization/v1/run.sh

```diff
@@ -0,0 +1,167 @@
+  data/callhome2 exp/ivectors_callhome2
+
+# Train a PLDA model on SRE, using callhome1 to whiten.
+# We will later use this to score iVectors in callhome2.
+run.pl exp/ivectors_callhome1/log/plda.log \
+  ivector-compute-plda ark:exp/ivectors_sre/spk2utt \
+  "ark:ivector-subtract-global-mean scp:exp/ivectors_sre/ivector.scp ark:- | transform-vec exp/ivectors_callhome1/transform.mat ark:- ark:- | ivector-normalize-length ark:- ark:- |" \
+  exp/ivectors_callhome1/plda || exit 1;
+
+# Train a PLDA model on SRE, using callhome2 to whiten.
+# We will later use this to score iVectors in callhome1.
+run.pl exp/ivectors_callhome2/log/plda.log \
+  ivector-compute-plda ark:exp/ivectors_sre/spk2utt \
+  "ark:ivector-subtract-global-mean scp:exp/ivectors_sre/ivector.scp ark:- | transform-vec exp/ivectors_callhome2/transform.mat ark:- ark:- | ivector-normalize-length ark:- ark:- |" \
+  exp/ivectors_callhome2/plda || exit 1;
+
+# Perform PLDA scoring on all pairs of segments for each recording.
+# The first directory contains the PLDA model that used callhome2
+# to perform whitening (recall that we're treating callhome2 as a
+# heldout dataset).  The second directory contains the iVectors
+# for callhome1.
+diarization/score_plda.sh --cmd "$train_cmd --mem 4G" \
+  --nj 20 exp/ivectors_callhome2 exp/ivectors_callhome1 \
+  exp/ivectors_callhome1/plda_scores
+
+# Do the same thing for callhome2.
+diarization/score_plda.sh --cmd "$train_cmd --mem 4G" \
+  --nj 20 exp/ivectors_callhome1 exp/ivectors_callhome2 \
+  exp/ivectors_callhome2/plda_scores
+
+# This performs unsupervised calibration using K-Means (K=2)
+# clustering on the scores.  The average of the centroids
+# is used as the estimated threshold.  Each partition is used
+# as a held-out dataset to compute the stopping criteria used
+# to cluster the other partition.
+diarization/compute_plda_calibration.sh --cmd "$train_cmd --mem 4G" \
+  --nj 20 exp/ivectors_callhome1 exp/ivectors_callhome2 exp/ivectors_callhome2/plda_scores
+
+diarization/compute_plda_calibration.sh --cmd "$train_cmd --mem 4G" \
+  --nj 20 exp/ivectors_callhome2 exp/ivectors_callhome1 exp/ivectors_callhome1/plda_scores
+
+# Cluster the PLDA scores using agglomerative hierarchical clustering,
+# using the thresholds discovered in the previous step.
+diarization/cluster.sh --cmd "$train_cmd --mem 4G" \
+  --nj 20 --threshold `cat exp/ivectors_callhome2/plda_scores/threshold.txt` \
+  exp/ivectors_callhome1/plda_scores exp/ivectors_callhome1/plda_scores
+
+diarization/cluster.sh --cmd "$train_cmd --mem 4G" \
+  --nj 20 --threshold `cat exp/ivectors_callhome1/plda_scores/threshold.txt` \
+  exp/ivectors_callhome2/plda_scores exp/ivectors_callhome2/plda_scores
+
+# Result using using unsupervised calibration
+# OVERALL SPEAKER DIARIZATION ERROR = 10.32 percent of scored speaker time  `(ALL)
+cat exp/ivectors_callhome1/plda_scores/rttm exp/ivectors_callhome2/plda_scores/rttm \
```

Comment:
I think we need to remove lines 134-159, along with the unsupervised calibration stuff. @danpovey felt that this method is too fragile (and my experience using it agrees that it isn't always reliable). Rather than trying to come up with a hasty replacement in this PR, I think it's better if we just remove it altogether, and compute thresholds using the held out datasets (e.g., find the optimal threshold for callhome1 and use it to cluster callhome2 and vice-versa).